src/ai_company/templates/builtins/dev_shop.yaml

```
@@ -0,0 +1,63 @@
+  variables:
+    - name: "company_name"
+      description: "Name of your company"
+      default: "Dev Shop"
+    - name: "budget"
+      description: "Monthly budget in USD"
+      var_type: "float"
+      default: 75.0
+
+  company:
+    type: "dev_shop"
+    budget_monthly: {{ budget | default(75.0) }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)